eclair-core/src/main/scala/fr/acinq/eclair/blockchain/electrum/ElectrumEclairWallet.scala

```diff
@@ -38,7 +38,7 @@ class ElectrumEclairWallet(val wallet: ActorRef, chainHash: BinaryData)(implicit
   override def makeFundingTx(pubkeyScript: BinaryData, amount: Satoshi, feeRatePerKw: Long) = {
     val tx = Transaction(version = 2, txIn = Nil, txOut = TxOut(amount, pubkeyScript) :: Nil, lockTime = 0)
     (wallet ? CompleteTransaction(tx, feeRatePerKw)).mapTo[CompleteTransactionResponse].map(response => response match {
-      case CompleteTransactionResponse(tx1, None) => MakeFundingTxResponse(tx1, 0)
+      case CompleteTransactionResponse(tx1, None) => MakeFundingTxResponse(tx1, 0, Satoshi(tx1.weight() * feeRatePerKw / 1000)) // TODO: output index is always 0? and check fee formula
```

Comment:
nitpick: using Transactions.weight2fee() would be cleaner, but modifying ElectrumWallet to return the fee would be even better